src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -153,6 +153,12 @@ exports[`Anchor icon label renders 1`] = `
   text-decoration: underline;
 }
 
+@media only screen and (max-width:768px) {
```

Comment:
I'm wondering why Anchor should be impacted by these changes?

Reply:
I think it's because Anchor uses a Box and the Box gap wasn't being handled responsively before. I will delve further into the old behavior to try to deconstruct what was happening before.

Reply:
I've updated Button, Anchor, and Menu to explicitly set `responsive={false}` for the Box containing the icon and label.